apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -324,6 +331,9 @@ var TopInstructions = React.createClass({
    * collapsed
    */
   getMinHeight(collapsed=this.props.collapsed) {
+    if (this.refs.containedLevel) {
+      return $('#containedLevel0').outerHeight() + 2 * CONTAINED_LEVEL_PADDING;
```

Comment:
looking at ContainedLevel, I would expect the heights of `#containedLevel0` and the `this.refs.containedLevel` container to be identical; can you instead use `getOuterHeight(this.refs.containedLevel)` here and below?